apps/backend/src/modules/groups/group.model.ts

```diff
@@ -0,0 +1,53 @@
+import { type RelationMappings } from "objection";
+
+import {
+	AbstractModel,
+	DatabaseTableName,
+} from "~/libs/modules/database/database.js";
+import { PermissionModel } from "~/modules/groups/permission.model.js";
+import { UserModel } from "~/modules/users/user.model.js";
+
+class GroupModel extends AbstractModel {
+	public key!: string;
+
+	public name!: string;
+
+	public permissions!: { id: number }[];
+
+	public users!: { id: number }[];
```

Comment:
Why can't we use just `number[]` instead of `{ id: number}[]` for `permissions` and `users`?
I just wanna know, cause belive you do it correct

Reply:
I think that Objection.js requires an object with an id field to properly handle many-to-many relationships.